src/itemlistformaction.cpp

```diff
@@ -213,6 +213,43 @@ bool ItemListFormAction::process_operation(Operation op,
 		}
 	}
 	break;
+	case OP_MARKITEMREAD: {
+		LOG(Level::INFO, "ItemListFormAction: marking item read at pos `%u'", itempos);
+		if (!visible_items.empty()) {
+			try {
+				const auto message_lifetime = v->get_statusline().show_message_until_finished(
+						_("Marking read flag for article..."));
+				// mark as undeleted
+				visible_items[itempos]
+				.first->set_deleted(false);
+				rsscache->mark_item_deleted(
+					visible_items[itempos]
+					.first->guid(),
+					false);
+				// toggle read
+				visible_items[itempos]
+				.first->set_unread(false);
+				v->get_ctrl()->mark_article_read(
+					visible_items[itempos]
+					.first->guid(),
+					false);
```

Comment:
The signature of the method is `void Controller::mark_article_read(const std::string& guid, bool read);`, so I would expect that this should be true to mark the item as read.